openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -133,21 +154,32 @@ def run(self, problem):
         self.quantities += list(iterkeys(incons))
         for name in incons:
             size = con_meta[name]['size']
-
-            # Bounds - double sided is supported
-            lower = con_meta[name]['lower']
-            upper = con_meta[name]['upper']
+            upper = np.zeros((size))
 
             # Sparsify Jacobian via relevance
             wrt = rel.relevant[name].intersection(param_list)
 
             if con_meta[name]['linear'] is True:
-                opt_prob.addConGroup(name, size, upper=upper, lower=lower,
-                                     linear=True, wrt=wrt,
-                                     jac=self.lin_jacs[name])
+                opt_prob.addConGroup(name, size, upper=upper, linear=True,
+                                     wrt=wrt, jac=self.lin_jacs[name])
             else:
-                opt_prob.addConGroup(name, size, upper=upper, lower=lower,
-                                     wrt=wrt)
+                opt_prob.addConGroup(name, size, upper=upper, wrt=wrt)
+
+        # TODO: Support double-sided constraints in openMDAO
+        # Add all double_sided constraints
+        #for name, con in iteritems(self.get_2sided_constraints()):
+            #size = con_meta[name]['size']
+            #upper = con.high * np.ones((size))
+            #lower = con.low * np.ones((size))
+            #name = '%s.out0' % con.pcomp_name
+            #if con.linear is True:
+                #opt_prob.addConGroup(name,
+                #size, upper=upper, lower=lower,
+                                     #linear=True, wrt=param_list,
+                                     #jac=self.lin_jacs[name])
+            #else:
+                #opt_prob.addConGroup(name,
```

Comment:
This pull request is reverting one of my whole commits.. Something weird happened here.